snapcraft/internal/lxd/_project.py

```diff
@@ -151,6 +155,30 @@ def _remote_mount(self, destination, source):
             'You can run the following command to enable it:\n'
             'echo Y | sudo tee /sys/module/fuse/parameters/userns_mounts')
 
+    def _setup_user(self):
+        # Setup user mirroring host user with sudo access
+        self._user = os.environ['USER']
+        self._project_folder = '/home/{}/build_{}'.format(
+            self._user, self._metadata['name'])
+        logger.debug('Setting up user {!r} in container'.format(self._user))
+        try:
+            self._container_run([
+                'useradd', self._user, '--create-home'])
+        except ContainerRunError as e:
+            # username already in use
+            if e.exit_code != 9:
+                raise e
+        self._container_run([
+            'usermod', self._user, '-o',
+            '-u', str(os.getuid()), '-G', 'sudo'])
+        self._container_run([
+            'chown', '{}:0'.format(os.getuid(), 0),
```

Comment:
You're calling `.format()` with two args, but only one replacement. I suspect you don't want the `0`, here. I'm also curious why we're using `0` for the group, here. Can you add a comment about that?